squash_wip.go

```diff
@@ -32,13 +24,24 @@ last commit must be a manual commit`)
 		mobExecutable()+" squash-wip --git-editor",
 		mobExecutable()+" squash-wip --git-sequence-editor",
 	)
-	silentgit("rebase", "-i", "--keep-empty", mergeBase)
+	git("rebase", "-i", "--keep-empty", mergeBase)
```

Comment:
having a sayInfo describing what is about to happen before the rebase might be helpful